pkg/util/scrubber/default.go

```diff
@@ -283,6 +283,22 @@ func ScrubYamlString(data string) (string, error) {
 	return string(res), nil
 }
 
+// ScrubJSON scrubs credentials from the given JSON by loading the data and scrubbing the object instead of the
+// serialized string, using the default scrubber.
+func ScrubJSON(data []byte) ([]byte, error) {
+	return DefaultScrubber.ScrubJSON(data)
+}
+
+// ScrubJSONString scrubs credentials from the given JSON string by loading the data and scrubbing the object instead of
+// the serialized string, using the default scrubber.
+func ScrubJSONString(data string) (string, error) {
+	res, err := DefaultScrubber.ScrubJSON([]byte(data))
```

Comment:
should it call `ScrubJSON()` instead ? 
```suggestion
	res, err := ScrubJSON([]byte(data))
```